### DISCUSSION TITLE
[Help]: Opacity effect with `slidesToScroll: 1` but two slides visible

### DISCUSSION THREAD

**Discussion by tobimori** [davidjerleke/embla-carousel#974] [Help]
### Summary

I've been trying for a while now, but I'm stuck at this point. I want the effect from the included CodeSandbox, (two slides visible at a time, others are hidden) but with `slidesToScroll: 1` instead of 2.

Any hint on what value to modify so this works?

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### CodeSandbox example

https://codesandbox.io/s/embla-carousel-opacity-vanilla-forked-lt2zs7?file=/src/js/index.ts

**Comment by davidjerleke** [comment]
Hi @tobimori,
With the current setup it's hard to achieve because it's based on the scroll progress in relation scroll snaps (not individual slides). I think what you want to achieve needs a different approach so maybe starting from scratch would be preferable.

**Reply by tobimori** [reply]
How would you do it, what values should I take a look at? I really like the "in progress" effect over the sudden change when the slide snap index changes.

**Reply by davidjerleke** [reply]
@tobimori could you share more details about what you want to achieve? Do you want every two slide opacity state to be in exact sync? For example, o you mean like this: If `slide 1` has an opacity of 0.5, `slide 2` should also have the same?

**Reply by tobimori** [reply]
No, they should be separate. You move 0 to -1, it fades out. 1 moves to 0 and stays fully visible. 2 moves to 1 and fades in.

**Reply by davidjerleke** [reply]
@tobimori unfortunately I'm not following. Is it possible for you to describe it more in detail? Maybe with images or something else that could help clarify what you want to achieve.

**Reply by tobimori** [reply]
This codesandbox now has the behaviour I want, but with a sudden change (since it uses the select event, no scrollProgress etc.)

https://codesandbox.io/s/embla-carousel-opacity-vanilla-forked-lt2zs7?file=/index.html

I want it to be a smooth transition, similar to the original opacity example.

**Reply by tobimori** [reply]
Sorry for the reminder, were you able to take a look at it? @davidjerleke

**Reply by davidjerleke** [reply]
@tobimori no worries about the reminder. Thanks for supporting this project. Unfortunately, I have a very busy period in my life so I barely have any spare time at all.

**Reply by tobimori** [reply]
totally understandable, i'll try to find a different solution for this for now.
